keyboards/shambles/keymaps/default/keymap.c

```diff
@@ -0,0 +1,10 @@
+#include "shambles.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	LAYOUT(
+		KC_ESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_BSPC, 
+		KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_ENT, KC_RCTL, KC_RALT, 
+		KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_RSFT, KC_UP, 
+		KC_Z, KC_X, KC_C, KC_V, KC_SPC, KC_B, KC_N, KC_M, KC_LEFT, KC_DOWN, KC_RGHT)
```

Comment:
Is this approximately the physical layout of the assembled keyboard? (Ctrl and Alt on the end of the home row seems strange.)

Reply:
Yes, this is. It's an alph28 with arrow keys and a nav cluster, but the nav cluster is a bit strange.